### PR TITLE
Fix tests for the Microwave task

### PR DESCRIPTION
**Pull request by woarewe**
Hi! Thank you very much for the task!
I was playing with Ruby track and found out that the task can be finished cause tests are broken.

https://exercism.org/tracks/ruby/exercises/microwave

### CODE REVIEW

File: exercises/practice/microwave/microwave_test.rb [hunk 1]
```diff
@@ -4,57 +4,39 @@
...
+  
   def test_one_minute_and_fifty_nine_seconds
-    skip
     assert_equal '01:59', Microwave.new(159).timer
   end
 
   def test_two_minutes
-    skip
-    assert_equal '02:00', Microwave.new(200).timer
+    assert_equal '02:00', Microwave.new(120).timer
```
Comment (kotp) [inline exercises/practice/microwave/microwave_test.rb:32]: This is not true for the task at hand.  Notice 159 being 1 minute and 59 seconds, and 200 should be 2 minutes.  The only part that is an exception to this is the less than 100 entered keeps the number of minutes.

File: exercises/practice/microwave/microwave_test.rb [hunk 2]
```diff
@@ -4,57 +4,39 @@
...
   end
 
-  def test_over_ten_minutes
-    # skip
-    assert_equal '10:01', Microwave.new(1001).timer
+  def test_ten_minutes_and_one_second
+    assert_equal '10:01', Microwave.new(601).timer
```
Comment (kotp) [inline exercises/practice/microwave/microwave_test.rb:36]: The same here, it should be that 1001 enters as 10 minutes and 1 second.

**Review by kotp — CHANGES_REQUESTED** [review]
Revert the test changes, but keep the removal of the skips.

File: exercises/practice/microwave/microwave_test.rb (hunk 1 above)
Comment (woarewe) [reply exercises/practice/microwave/microwave_test.rb:32]: So, now I got the idea. Thank you very much

**Review by kotp — APPROVED** [review]
Thank you @woarewe !

**Pull request merged**